sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -652,11 +676,11 @@ def readall(self) -> T:
         :rtype: T
         """
         stream = BytesIO()
-        self.readinto(stream)
+        self.readinto(stream)  # type: ignore [arg-type]
```

Comment:
`**azure\storage\blob\_download.py:679: error: Argument 1 to "readinto" of "StorageStreamDownloader" has incompatible type "BytesIO"; expected "IO[str]"  [arg-type]**`